app/src/main/java/ar/com/wolox/android/example/utils/UserSession.kt

```diff
@@ -12,11 +12,27 @@ class UserSession @Inject constructor(private val sharedPreferencesManager: Shar
     // application, but we should add a check in case Android decides to kill the application
     // and return to a state where this isn't initialized.
     var username: String? = null
-        get() = field ?: sharedPreferencesManager[Extras.UserLogin.USERNAME, null].also {
-            field = it
+    var password: String? = null
+    var loginOk: Boolean = false
+
+    // Para setear u obtener el username guardado en el sharedPreferences
+    fun get_Username() = username ?: sharedPreferencesManager[Extras.UserLogin.USERNAME, null].also {
```

Comment:
we don't usually use underscore when declaring functions. Change them to something like "username", replicating the code that was written before in this file